src/elm/Favorites.elm

```diff
@@ -0,0 +1,84 @@
+{--
+Copyright (c) 2019 Target Brands, Inc. All rights reserved.
+Use of this source code is governed by the LICENSE file in this repository.
+--}
+
+
+module Favorites exposing (ToggleFavorite, isFavorited, starToggle, toFavorite, updateFavorites)
+
+import Html exposing (Html)
+import Html.Events exposing (onClick)
+import List.Extra
+import RemoteData exposing (RemoteData(..), WebData)
+import Svg.Attributes
+import SvgBuilder exposing (star)
+import Util
+import Vela exposing (CurrentUser, Org, Repo)
+
+
+{-| ToggleFavorite : takes org and maybe repo and toggles its favorite status them on Vela
+-}
+type alias ToggleFavorite msg =
+    Org -> Maybe Repo -> msg
+
+
+
+-- VIEW
+
+
+{-| starToggle : takes org repo msg and favorited status and renders the favorites toggle star
+-}
+starToggle : Org -> Repo -> ToggleFavorite msg -> Bool -> Html msg
+starToggle org repo toggleFavorite favorited =
+    star
```

Comment:
this probably needs to get marked up better for accessibility

Reply:
done